src/main/java/com/example/codeE/service/docker/DockerImpl.java

```diff
@@ -50,7 +54,7 @@ public String createContainer(String dockerfilePath, CodeExercise codeExercise)
         docker.replaceFile(
                 containerId,
                 FileHelper.replaceTemplateFile(
-                        "./template/java/MainTemplate.txt",
+                        "./template/java/MainJavaTemplate.txt",
```

Comment:
Tell me why you renamed the file to MainJavaTemplate. Suppose if this path file is used in the common function, instead of just changing the package name java to another language, we must also change the file name?

Reply:
This is my mistake, I did not see the package "java". but this is not a big issue, You can merge it first